CHANGELOG.md

```diff
@@ -1,18 +1,20 @@
 ## 0.25.0 (April 29, 2021)
 
 BREAKING CHANGES:
+* d/tfe_workspace: Due to the addition of remote controlled state consumer functionality, using this data source requires using the provider with Terraform Cloud or an instance of Terraform Enterprise at least as recent as v202104-1 ([#292](https://github.com/hashicorp/terraform-provider-tfe/pull/292))
 * d/tfe_workspace: Removed deprecated `external_id` attribute. Use `id` instead ([#295](https://github.com/hashicorp/terraform-provider-tfe/pull/295))
 * d/tfe_workspace_ids: Removed deprecated `external_ids` attribute. Use `ids` instead ([#295](https://github.com/hashicorp/terraform-provider-tfe/pull/295))
 * r/tfe_workspace: Removed deprecated `external_id` attribute. Use `id` instead ([#295](https://github.com/hashicorp/terraform-provider-tfe/pull/295))
+* r/tfe_workspace: Due to the addition of remote controlled state consumer functionality,Using this resource requires using the provider with Terraform Cloud or an instance of Terraform Enterprise at least as recent as v202104-1 ([#292](https://github.com/hashicorp/terraform-provider-tfe/pull/292))
```

Comment:
Oh, I just noticed this lil typo friend! nbd to fix later if needed, though.

```suggestion
* r/tfe_workspace: Due to the addition of remote controlled state consumer functionality, using this resource requires using the provider with Terraform Cloud or an instance of Terraform Enterprise at least as recent as v202104-1 ([#292](https://github.com/hashicorp/terraform-provider-tfe/pull/292))
```

Reply:
good catch! just fixed it 🙌🏼